CONTRIBUTING.md

```diff
@@ -77,11 +79,14 @@ the [development mailing list](http://dev.list.galaxyproject.org/).
   request](https://help.github.com/articles/creating-a-pull-request/)
   with these changes. You pull request message ideally should include:
 
-   * A description of why the change should be made.
+   * A description of why the changes should be made.
 
-   * A description of implementation of the change.
+   * A description of the implementation of the changes.
 
-   * A description of how to test the change.
+   * A description of how to test the changes.
+
+* The pull request should pass all the continuous integration tests which are
```

Comment:
Yeah - I requested this specifically because I had the same concern. I am cool with the ``should`` language - it means we don't have to wait but if merge it better still pass the tests. I think that will be enough to cause whoever merged it to followup with a fix post-merge if something broke because it was merged too quickly.